src/platform/windows/window.rs

```diff
@@ -622,6 +622,14 @@ pub struct WindowWrapper(HWND, HDC);
 // https://github.com/retep998/winapi-rs/issues/396
 unsafe impl Send for WindowWrapper {}
 
+pub unsafe fn adjust_size(
+    (x, y): (u32, u32), style: DWORD, ex_style: DWORD,
```

Comment:
Ooh, I never knew you could destructure a tuple in the function parameters. Thanks for teaching me something new today!